app/daemon.js

```diff
@@ -97,11 +107,13 @@ export function getAPIVersion () {
 
 /**
  * startIPFSDaemon will start IPFS go daemon, if installed.
- * return a promise with child process of IPFS daemon
+ * return a promise with child process of IPFS daemon.
+ * The daemon always has 2 options, one to ensure that the repo is initalized
+ * the other one to ensure that the api endpoint is the right multiaddr
  */
 export function startIPFSDaemon () {
   return new Promise((resolve, reject) => {
-    const ipfsProcess = spawnIPFSCommand('daemon', `--api=${global.IPFS_MULTIADDR_API}`)
+    const ipfsProcess = spawnIPFSCommand('daemon', '--init', `--api=${global.IPFS_MULTIADDR_API}`)
```

Comment:
Do we need to pass `--api=${global.IPFS_MULTIADDR_API}` here? What about the other addresses?

Reply:
Yes, it is better to!
The others are configured on another method. AFAIK this is not documented and I was not able to find the other args